queries/elm/highlights.scm

```diff
@@ -0,0 +1,88 @@
+; Keywords
+[
+    "if"
+    "then"
+    "else"
+    "let"
+    "in"
```

Comment:
isn't `in` more a `@keyword.operator` and `let` not conditional at all?

Reply:
i believe these are control flow keywords. I'm currently just learning elm so i'm not 100% sure yet.
of is used with a case (pattern matching), much like
```elm
case some_val of
 ```

Reply:
yeah, that's conditional :smile:

Reply:
To chime in as an actual Elm user, `case` and `of` are definitely conditionals and a `let`/`in` pair is definitely not. I'm still familiarizing myself with the types treesitter provides, but within the context of this PR a `let`/`in` pair is much more inline with keywords.

Reply:
fixed by adding to keywords.